add-to-org.rb

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env ruby
+# add USERNAME as member to ORGANIZATION and optionally to TEAM
+# assumes you created and stored the appropriate OAuth token as an ENV variable called GITHUBUSER_TOKEN
+
+require 'octokit'
+require 'optparse'
+
+# Make sure arguments are specified
+ARGV << '-h' if ARGV.empty?
```

Comment:
This is an interesting hack. A bit surprised `OptionParser` doesn't  support the empty case.